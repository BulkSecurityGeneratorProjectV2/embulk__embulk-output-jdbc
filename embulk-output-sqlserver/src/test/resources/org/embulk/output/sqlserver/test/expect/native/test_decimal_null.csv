ID:string,value:long
1,
2,9999
