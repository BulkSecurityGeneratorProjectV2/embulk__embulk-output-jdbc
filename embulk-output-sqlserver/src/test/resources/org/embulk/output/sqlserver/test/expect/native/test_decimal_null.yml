table: TEST_DECIMAL
mode: insert_direct
insert_method: native
