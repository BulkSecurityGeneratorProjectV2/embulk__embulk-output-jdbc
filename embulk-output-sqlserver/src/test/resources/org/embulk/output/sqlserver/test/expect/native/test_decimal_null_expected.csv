1   ,NULL
2   ,9999.0000
